### DISCUSSION TITLE
Is it possible to privatize access?

### DISCUSSION THREAD

**Discussion by curiosport** [uvdesk/community-skeleton#661] [General]
I would like to use this system for a specific group of people but I want only this group and myself to be able to see the content, anyone who has the URL to a ticket first must log in, so neither search engines nor anyone except those who are registered will be able to use the system, do you have that feature or should I develop it on my own?

**Comment by komal-sh-27** [comment]
@curiosport 

I think your requirement fullfill from the agent option in groups and permission tab, also agent can view tickets shared group permissions only. 

![image](https://github.com/uvdesk/community-skeleton/assets/63539748/88468893-8518-49b5-a3a2-1c47bcb667f2)

![image](https://github.com/uvdesk/community-skeleton/assets/63539748/fa2ad36f-0b11-4c10-b7a2-dbdf13582375)


If yes then you can try it and let us know is it fullfill your requirement or not.

You can follow this blog for how to add an agent on helpdesk:

https://support.uvdesk.com/en/blog/uvdesk-how-to-add-support-agents

**Reply by curiosport** [reply]
thank you very much for the data, it was very useful.

**Comment by komal-sh-27** [comment]
Alright, thanks for the reply. At this moment we are closing this discussion.
If you have any further any query then let us know and also you can reply on same thread.
